app/chat/tasks.py

```diff
@@ -0,0 +1,48 @@
+from django.conf import settings
+
+from app.redis_service import RedisService
+from celery import app, task
+from celery.utils.log import get_task_logger
+
+from mattermostdriver import Driver
+
+mm_driver = Driver({
+    'url': settings.CHAT_URL,
```

Comment:
is there a doc on how to setup mattermost locally for local gitcoin dev? is it just another docker command?